xrefcheck.nix

```diff
@@ -4,11 +4,8 @@
 
 { linux ? false, linux-static ? false, windows ? false }:
 let
-  sources = import ./nix/sources.nix;
-  haskell-nix = import sources."haskell.nix" {
-    sourcesOverride = { hackage = sources."hackage.nix"; stackage = sources."stackage.nix"; };
-  };
-  nixpkgs = import sources.nixpkgs haskell-nix.nixpkgsArgs;
+  nixpkgs = (import ./ci.nix).pkgs;
+  src = (import ./ci.nix).project-src;
```

Comment:
Somehow I'm not really keen on importing stuff from `ci.nix` when it's just for building the project and not necessarily for CI, but that's just bikeshedding.

Reply:
Yeah, I thought that it's weird too, I just didn't want to bother too much about it